05-EventAggregator/src/PrismSample/ViewModels/DataEntryPageViewModel.cs

```diff
@@ -34,5 +35,14 @@ private void OnSubmitTapped()
         {
             _navigationService.GoBackAsync ();
         }
+
+        public override void OnNavigatedTo(INavigationParameters parameters)
+        {
+            base.OnNavigatedTo(parameters);
+            if (parameters.ContainsKey(NavigationParameterKeys.CurrentIsFunValue))
+            {
+                IsFun = (bool)parameters[NavigationParameterKeys.CurrentIsFunValue];
```

Comment:
this is a deprecated method. Use `parameters.GetValue<bool>("CurrentIsFunValue");`